crypto/evp/m_sigver.c

```diff
@@ -701,8 +701,10 @@ int EVP_DigestVerify(EVP_MD_CTX *ctx, const unsigned char *sigret,
         }
     } else {
         /* legacy */
-        if (ctx->pctx->pmeth != NULL && ctx->pctx->pmeth->digestverify != NULL)
-            return ctx->pctx->pmeth->digestverify(ctx, sigret, siglen, tbs, tbslen);
+        if (pctx == NULL)
+            return -1;
```

Comment:
Can you please move the `pctx == NULL` check outside of this if/else and fail out? There is no way the EVP_DigestVerifyFinal() call would succeed with `pctx == NULL`.

You can also add  `ERR_raise(ERR_LIB_EVP, EVP_R_INITIALIZATION_ERROR);` before returning the error.

Reply:
Moved the check to NULL above